libs/design/src/atoms/form/select/select/select.component.spec.ts

```diff
@@ -0,0 +1,234 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+
+import { DaffNativeSelectComponent } from './select.component';
+
+@Component({
+  template: `<select daff-native-select [formSubmitted]="formSubmittedValue" [formControl]="formControlValue"></select>`
+})
+
+class WrapperComponent {
+  formSubmittedValue: boolean;
+  formControlValue: FormControl;
+}
+
+describe('DaffNativeSelectComponent', () => {
+  let wrapper: WrapperComponent;
+  let fixture: ComponentFixture<WrapperComponent>;
+  let stubFormControl;
+  let stubFormSubmitted: boolean;
+  let component: DaffNativeSelectComponent;
+  let hostNativeElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ 
+        DaffNativeSelectComponent,
+        WrapperComponent
+       ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    stubFormControl = new FormControl();
+    stubFormSubmitted = false;
+
+    fixture = TestBed.createComponent(WrapperComponent);
+    wrapper = fixture.componentInstance;
+    wrapper.formControlValue = stubFormControl;
+    wrapper.formSubmittedValue = stubFormSubmitted;
+    fixture.detectChanges();
+
+    component = fixture.debugElement.query(By.css('[daff-native-select]')).componentInstance;
+  });
+
+  it('should create', () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  describe('daff-native-select',() => {
```

Comment:
I don't think you need this describe. It's already within the `describe('DaffNativeSelectComponent')`, so this is repetitive.

Reply:
Removed aa6853d

Reply:
fixed aa6853d